squad/core/tasks/__init__.py

```diff
@@ -511,3 +512,11 @@ def cleanup_build(build_id):
     build = Build.objects.get(pk=build_id)
     BuildPlaceholder.objects.create(project=build.project, version=build.version)
     build.delete()
+
+
+@celery.task
+def update_build_patch_url(build_id):
```

Comment:
This will be sent to celery's default queue, could you please add this to `squad/settings.py > CELERY_TASK_ROUTES`, I think `core_quick` fits this nicely

Reply:
done